src/mdframe/schema.py

```diff
@@ -0,0 +1,126 @@
+schema = {
+  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
Like mentioned in my other comment you might want to store this as an actual JSON file committed into the repository and read that JSON when the `Config` instance is being constructed.

Reply:
done 🎸